libs/cvr-fixture-generator/src/cli/generate/main.ts

```diff
@@ -261,9 +256,29 @@ export async function main(
         );
 
         const layout = assertDefined(layouts[i]);
-        fs.writeFileSync(layoutFilePath, JSON.stringify(layout));
+        const layoutFileContents = JSON.stringify(layout);
+        fs.writeFileSync(layoutFilePath, layoutFileContents);
+
+        const imageHash = sha256(fs.readFileSync(imageFilePath));
+        const layoutFileHash = sha256(layoutFileContents);
+        populateImageAndLayoutFileHashes(
+          assertDefined(castVoteRecord.BallotImage[i]),
+          { imageHash, layoutFileHash }
+        );
       }
     }
+
+    const castVoteRecordReport: CVR.CastVoteRecordReport = {
+      ...reportMetadata,
+      CVR: [castVoteRecord],
+    };
+    fs.writeFileSync(
+      join(
+        castVoteRecordDirectory,
+        CastVoteRecordExportFileName.CAST_VOTE_RECORD_REPORT
+      ),
+      JSON.stringify(castVoteRecordReport)
+    );
```

Comment:
As in `libs/backend`, I had to change the order of operations here, exporting the `cast-vote-record-report.json` after the images and layout files (if applicable)